docs/getting-started/development-environment.rst

```diff
@@ -178,7 +178,105 @@ images are having problems building.
 Linux
 ~~~~~
 
-Want to write it? Contact us on the chat!
+Useful packages to install:
+
+.. code-block:: sh
+
+        git:      $ apt install git-all
+
+        zoom:     https://support.zoom.us/hc/en-us/articles/204206269-Installing-or-updating-Zoom-on-Linux#h_adcc0b66-b2f4-468b-bc7a-12c182f354b7
+
+        discord:  https://doc.ubuntu-fr.org/discord
+        
+        VS code:  https://code.visualstudio.com/download
+
+If during the installation you encounter broken packages, try the follwoing command:
+
+.. code-block:: sh
+        $ apt --fix-broken install
+
+
+**Python**
+
+Invenio is developed using Python and JavaScript. 
+If you want to check which version of Python you have, try the following:
+
+.. code-block:: sh
+        # Check the system Python version
+        $ python --version
+
+        # Check the Python 2 version
+        $ python2 --version
+
+        # Check the Python 3 version
+        $ python3 --version
+
+To install Python 3.8 type the following commands:
+
+.. code-block:: sh
+
+        $ sudo apt-get update
+        $ sudo apt-get install python3.8 python3-pip
+
+In the follwoing list you can check if your system has the necessary
+requirements. We highly recommend that you install ``pyenv`` and ``nvm`` - both tools manage 
+version of python and node respectively.
+https://inveniordm.docs.cern.ch/install/requirements/
+
+After installing docker you might have some errors related to permissions, try the following command: 
+
+.. code-block:: sh
+        $ sudo chmod 666 /var/run/docker.sock
+
+To check if you have the latest version of node installed type the following commands:
+
+.. code-block:: sh
+        $ sudo npm cache clean -f
+        $ sudo npm install -g n
+        $ sudo n stable
+
+General tools/packages useful during development:
+
+.. code-block:: sh
+        $ apt-get install sqlitebrowser        # UI for SQLite
```

Comment:
I would harmonize to remove `sudo` if not needed, or add it everywhere if needed.